earthquakes.py

```diff
@@ -31,31 +31,73 @@ def get_data():
 
     # We need to interpret the text to get values that we can work with.
     # What format is the text in? How can we load the values?
-    return ...
+    return json.loads(text)
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
-    return ...
+    return data.get("metadata").get("count")
 
 
 def get_magnitude(earthquake):
     """Retrive the magnitude of an earthquake item."""
-    return ...
+    return [dict.get("properties").get("mag") for dict in data.get("features")]
 
 
 def get_location(earthquake):
     """Retrieve the latitude and longitude of an earthquake item."""
     # There are three coordinates, but we don't care about the third (altitude)
-    return ...
+    list_of_coords = [dict.get("geometry").get("coordinates") for dict in data.get("features")]
+    lat_lon_only = [coords[0:2] for coords in list_of_coords]
+    return lat_lon_only
 
 
 def get_maximum(data):
     """Get the magnitude and location of the strongest earthquake in the data."""
-    ...
+    #this builds an element that has magnitude, lat, lon, altitude. I used [0:3] to discard altitude. I am also using reverse Sorted to sort by the first element only (which is mag) so that I can extract lat lat afterwards
+    mag_lat_lon = sorted([(([dict.get("properties").get("mag")]+dict.get("geometry").get("coordinates"))[0:3]) for dict in data.get("features")], reverse = True)
```

Comment:
I see what you mean, this is a bit convoluted. You could make it easier to read by splitting it into two steps and using the functions above. Perhaps
```suggestion
    mag_lat_lon = zip(get_magnitude(data["features"]), get_location(data["features"])
    mag_lat_lon = sorted(mag_lat_lon), reverse = True)
```
This expression is now smaller, so you could even do it in one line, perhaps, without the intermediate variable.

Note how you can combine the elements of the two lists of results using `zip`. Reusing the functions also simplifies the expression by "hiding" the comprehensions.